src/tests/ftest/scrubber/performance.py

```diff
@@ -0,0 +1,73 @@
+"""
+  (C) Copyright 2018-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from ior_test_base import IorTestBase
+from ior_utils import IorMetrics
+
+
+class TestWithScrubberPerf(IorTestBase):
+    # pylint: disable=too-many-nested-blocks
+    """Basic Scrubber Test
+
+    :avocado: recursive
+    """
+
+    def test_scrubber_performance(self):
+        """JIRA ID: DAOS-7372
+        Check IOR performance with checksum enabled.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium
+        :avocado: tags=scrubber
+        :avocado: tags=test_scrubber_performance
+        """
+        write_flags = self.params.get("write_flags", self.ior_cmd.namespace)
+        read_flags = self.params.get("read_flags", self.ior_cmd.namespace)
+        self.add_pool()
+        self.add_container(self.pool)
```

Comment:
Not a blocker, but we want to move toward these functions because implicitly letting a function set `self.*` variables leads to a lot of problems. The idea is that (1) we should minimally set `self.*` variables, and (2) individual test cases should set those if needed, not utility/helper functions
```suggestion
        self.pool = self.get_pool()
        self.container = self.get_container(self.pool)
```